src/edit_command_manager.ts

```diff
@@ -54,19 +54,22 @@ export class EditCommandManager {
     }
 
     //WORKS
+    // There could be an error if the line the user asks to delete is "if (hello > 5)". Good to do checks for
```

Comment:
Was this something you built in? or was my delete line function originally like this? cause i dont think it was